modules/osd-rhoam.adoc

```diff
@@ -1,11 +1,24 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/available-services.adoc
+// * adding_service_cluster/rosa-available-services.adoc
 
 [id="osd-rhoam_{context}"]
-
 = Red Hat OpenShift API Management
 
+The Red Hat OpenShift API Management (OpenShift API Management) service is available as an add-on to your {product-title} on AWS cluster. OpenShift API Management is a managed API traffic control and API program management solution that is based on the 3scale API Management platform and includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.
+
+This OpenShift API Management entitlement provides:
```

Comment:
Is _entitlement_ the right word here? SSG is particular about not using _entitlement_ when you mean _subscription_. I do not grasp the finer distinctions, so just a question.